README.md

```diff
@@ -31,7 +31,32 @@ where
   (last_success_login > now() - interval '30 days')
   and last_success_login is not null;
 ```
+## Examples
```

Comment:
Done

Reply:
@paulsewardasc Did you push up the new commit yet? I don't see it in the `Commits` list yet.

Reply:
It is here, do I need to do something else? It may not have been here last night.

https://github.com/paulsewardasc/steampipe-plugin-snowflake/tree/resource_monitors